preconditions/build.gradle

```diff
@@ -0,0 +1,7 @@
+apply from: "${rootDir}/gradle/publish.gradle"
+
+dependencies {
+    implementation project(':safe-logging')
+    implementation 'com.google.code.findbugs:jsr305'
```

Comment:
what's this used for?

Reply:
nothing! about to remove, though looks like dfox beat me.